scripts/known_good.json

```
@@ -7,15 +7,15 @@
             "sub_dir": "Vulkan-Headers",
             "build_dir": "Vulkan-Headers/build",
             "install_dir": "Vulkan-Headers/build/install",
-            "commit": "v1.3.302"
+            "commit": "v1.4.303"
```

```suggestion
            "commit": "6a74a7d65cafa19e38ec116651436cce6efd5b2e"
```

Should fix the tests now

---

Not even needed, the reference behind the 1.4.303 tag has been updated!